ansible_base/authentication/authenticator_plugins/ldap.py

```diff
@@ -583,6 +589,7 @@ def get_or_build_user(self, username, ldap_user):
         """
         user, _authenticator_user, created = get_or_create_authenticator_user(
             username.lower(),
+            ldap_user.attrs.data.get(ldap_user.settings.USER_ATTR_MAP['email'], ""),
```

Comment:
@zkayyali812 do you want to fix this one too

Reply:
Yes, I'll fix this one too

Reply:
Done